itou/templates/employee_record/missing_employee.html

```diff
@@ -0,0 +1,96 @@
+                                    Si vous souhaitez bénéficier d’une aide au poste, vous devez d’abord obtenir un PASS IAE en effectuant une nouvelle embauche, puis créer sa fiche salarié.
+                                </p>
+                            {% else %}
+                                <p>Ce salarié a {{ approvals_data|length }} PASS IAE disponible{{ approvals_data|length|pluralizefr }}</p>
+
+                                {% for approval, job_application, approval_case, employee_record in approvals_data %}
+                                    <hr>
+                                    <p>
+                                        <strong>{{ approval.number }}</strong> du {{ approval.start_at|date:'d/m/Y' }} au {{ approval.end_at|date:'d/m/Y' }}
+                                    </p>
+                                    {% if approval_case == MissingEmployeeCase.FUTURE_HIRING %}
+                                        <p>
+                                            La création de fiche salarié sera possible à compter de la date d’embauche : {{ job_application.hiring_start_at|date:'d/m/Y' }}.
+                                        </p>
+                                    {% elif approval_case == MissingEmployeeCase.EXISTING_EMPLOYEE_RECORD_OTHER_COMPANY %}
+                                        <p>
+                                            Une fiche salarié existe déjà pour ce PASS IAE dans une autre structure avec la même convention : {{ employee_record.job_application.to_company.kind }} {{ employee_record.job_application.to_company.display_name }} - SIRET {{ employee_record.job_application.to_company.siret|format_siret }}
+                                        </p>
+                                        <p>
+                                            Si vous souhaitez visualiser cette fiche salarié, vous devez vous connecter au tableau de bord de cette structure.
+                                        </p>
+                                        <p>
+                                            Seule la première embauche avec ce PASS IAE, nécessite une transmission de la fiche salarié.
+                                            <br>
+                                            En cas de besoin vous pouvez ajouter un nouveau contrat depuis votre espace Extranet IAE 2.0 de l’ASP
+                                        </p>
+                                    {% elif approval_case == MissingEmployeeCase.EXISTING_EMPLOYEE_RECORD_SAME_COMPANY %}
+                                        <p>
+                                            Une fiche salarié existe déjà pour ce PASS IAE : <a href="{% url 'employee_record_views:summary' employee_record.pk %}">Voir la fiche salarié</a>.
+                                        </p>
+                                        <p>
+                                            Seule la première embauche avec ce PASS IAE, nécessite une transmission de la fiche salarié.
```

Comment:
```suggestion
                                            Seule la première embauche avec ce PASS IAE nécessite une transmission de la fiche salarié.
```
me semble plus correct ?